.circleci/config.yml

```
@@ -25,21 +25,48 @@ commands:
           command: docker-compose up -d
 
 jobs:
-  build_and_test:
+  build:
     working_directory: ~/repo
     docker:
       - image: cimg/node:18.16.0
       - image: cimg/postgres:14.1
```

at the top of this file you can add something like this:

```yaml
defaults: &defaults
  working_directory: ~/repo
  docker:
    - image: cimg/node:18.16.0
    - image: cimg/postgres:14.1
```

and here do:
```suggestion
    <<: *defaults
```

This way you don't have to repeat this part every time. If you need to use other images or working directory you just define the key the same way as right now - it will override given key from `defaults`

---

of course this is optional - some devs don't like using `defaults` as for them it's not a readable as it could be